clusterloader2/cmd/clusterloader.go

```diff
@@ -149,7 +149,7 @@ func completeConfig(m *framework.MultiClientSet) error {
 		}
 	}
 	if len(clusterLoaderConfig.ClusterConfig.MasterIPs) == 0 {
-		masterIPs, err := util.GetMasterIPs(m.GetClient(), corev1.NodeExternalIP)
+		masterIPs, err := util.GetMasterIPs(m.GetClient(), NodeExternalIP)
```

Comment:
Why introducing another, redundant constant instead of using corev1.NodeExternalIp?

Reply:
This is the error we are getting when used the corev1 constants...
 
perf-tests/clusterloader2/cmd/clusterloader.go:153:54: cannot use corev1.NodeExternalIP (constant "ExternalIP" of type v1.NodeAddressType) as v1.NodeAddressType value in argument to util.GetMasterIPs